taxonomies/categories.txt

```diff
@@ -1417,41 +1417,304 @@ agribalyse_proxy_food_name:en:Beer, dark
 agribalyse_proxy_food_name:fr:Bière brune
 
 <en:Beers
-en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales, IPA beers
-es:Cerveza IPA, IPA
+en:Packaged beers, Beers in a package, Packed beers
+lt:Supakuotas alus, Alus pakuotėje, Alus dėžėje
+
+################################# ALES #################################
+
+<en:Beers
+en:Ales, Ale
+lt:Elis
+wikidata:en:Q208385
+
+<en:Ales
+en:Pale ales, Pale ale
+
+############ IPA ############
+
+<en:Pale ales
 ca:Cervesa IPA, IPA
+en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales
+es:Cerveza IPA, IPA
+fr:Bières IPA, Bieres IPA, Bière ipa
 xx:India Pale Ale (IPA)
 wikidata:en:Q1543214
 
 <en:India Pale Ale (IPA)
-en:Milkshake IPA, Milkshake India Pale Ale
-wikidata:en:Q103351397
+xx:Amber IPA
```

Comment:
For categories, we always put an en: entry as the canonical entry, and then optionally we can add a xx: category that matches all languages

So all xx: entries below should be converted to en:, and we can add a corresponding xx: entry below.